lib/ssl/src/ssl_server_session_cache.erl

```diff
@@ -62,8 +62,8 @@
 %%%===================================================================
 
 -spec start_link(pid(), map()) -> {ok, Pid :: pid()} |
-                      {error, Error :: {already_started, pid()}} |
-                      {error, Error :: term()} |
+                      {error, Error1 :: {already_started, pid()}} |
```

Comment:
@IngelaAndin same here

Reply:
Hum .. do we really need this name here can we no just remove it instead!?